python/plugins/processing/gui/ParametersPanel.py

```diff
@@ -204,3 +204,9 @@ def buttonToggled(self, value):
             for button in list(self.iterateButtons.values()):
                 if button is not sender:
                     button.setChecked(False)
+
+    @staticmethod
+    def tr(string, context=''):
+        if context == '':
+            context = 'ParametersPanel'
+        return QCoreApplication.translate(context, string)
```

Comment:
Why not simply:
```python
    @staticmethod
    def tr(string, context='ParametersPanel'):
        return QCoreApplication.translate(context, string)
```

Reply:
Why Implement it at all, this is a QWidget subclass and already has `tr()` method

Reply:
That's the why part in the description above. Currently it is not translating (at least in my system).

Reply:
I just checked with a nightly Windows build and the problem is there too. Maybe it is a context issue, however I don't see why. Anyway, currently the tr from QWidget is not translating the strings. I could remove the tr from this class and use explicit QCoreApplication.translate calls as it works that way.